test/test-hapi.js

```diff
@@ -1,22 +1,32 @@
+const nodeVersionIsSupported = Number(process.versions.node.split('.')[0]) > 10;
+
 const test = require('ava');
 const supertest = require('supertest');
-const Hapi = require('@hapi/hapi');
 const Agenda = require('agenda');
 
 const agenda = new Agenda().database('mongodb://127.0.0.1/agendash-test-db', 'agendash-test-collection');
+let request;
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable ava/no-unknown-modifiers */
+const testType = nodeVersionIsSupported ? 'serial' : 'skip';
```

Comment:
Nice idea! 👏

Reply:
Did you notice two eslint rules? It's the only way when using `xo`. I've never seen this kind of stuff before. :)

Reply:
Well linters are here just to guide us, we shouldn't blindly follow them.

An alternative I considered was to somehow conditionally return early in each test, but I opted in the bash if/else in Travis instead as it felt cleaner since we had only two test files.